### DISCUSSION TITLE
Why Github Authentication Extension(vscode.github-authentication) is redirecting to github.com even after setting up "github-enterprise.uri" in extension settings.

### DISCUSSION THREAD

**Discussion by parthjeet** [microsoft/vscode-discussions#562] [Extension Development QnA] [answered]
I have configured the "github-enterprise.uri" with my work's github url, but still when I'm trying to sign in using github, the extension is redirecting to github.com in browser for authentication.

**Comment by alefragnani** [comment]
Hi @parthjeet ,

I suppose you are using the GitHub Pull Requests and Issues extension. If that's the case, based on the extension's wiki (https://github.com/Microsoft/vscode-pull-request-github/wiki#version-0500-and-lower), this is the right setting to be used. But, based on this open issue https://github.com/microsoft/vscode-pull-request-github/issues/4660, it seems to have some issue with it. If you look for other issues about _enterprise_, you will find a few open (https://github.com/microsoft/vscode-pull-request-github/issues?q=is%3Aissue+is%3Aopen+enterprise)

I suggest you to interact on that issues, so the team could take a look at your scenario in order to fix it.

Hope this helps

**Reply by parthjeet** [reply]
Hi @alefragnani 
No, the concern is not about the Github pull request extension but the in-built Github extension which provides functionalities like "Publish to Github".

**Reply by alefragnani** [reply, marked as answer]
Ah, right. Are you developing an extension relying on it (using the `AuthenticationProvider` API)? 

I suppose the GitHub Issues and Pull Requests extension uses the same API so, if they are have issues, others may have as well. I would suggest you to post an issue in VS Code repo itself, if no solution/direction is provided here, but I noticed you already did https://github.com/microsoft/vscode/issues/179813. 

Let's see if someone else had the same difficulties and fixed, or if this is an issue in the extension, that has to be fixed in VS Code itself.
